src/invidious/yt_backend/extractors.cr

```diff
@@ -175,7 +175,10 @@ private module Parsers
       # TODO change default value to nil
 
       subscriber_count = item_contents.dig?("subscriberCountText", "simpleText")
-
+      channel_handle = nil
+      if subscriber_count.try &.as_s.starts_with?("@")
+        channel_handle = subscriber_count.try &.as_s
+      end
```

Comment:
Note that I couldn't include the line above in the suggestion because github.

You'll also have to remove the `.as_s` in the next "paragraph".

```suggestion
      subscriber_count = item_contents.dig?("subscriberCountText", "simpleText").try &.as_s
      channel_handle = subscriber_count if (subscriber_count.try &.starts_with? "@")
```